### DISCUSSION TITLE
More tiers for github sponsors?

### DISCUSSION THREAD

**Discussion by cesarbiods** [denoland/deno#8767] [Ideas]
I know it's technically not related to the repo here but I requested this in the past and received no response. I love the work you guys are doing and would like to support you financially in your endeavors but I (and I'm sure others as well) would like to see more tier options in github sponsors, specifically an amount lower than $10. Regardless, keep up the good work 🙂.

**Comment by ry** [comment]
Ok I added a $1 tier - thanks

**Comment by cesarbiods** [comment]
That works! Just signed up, thanks for the flexibility.
